scripts/mixins/industrialcraft2.zs

```diff
@@ -0,0 +1,424 @@
+#{targets: "ic2.core.crop.cropcard.CropHops"}
+zenClass MixinCropHops {
+    #mixin ModifyConstant
+    #{
+    #    method: "getGrowthDuration",
+    #    constant: {intValue: 600}
+    #}
+    function growFaster(value as int) as int {
+        return 100;
+    }
+}
+
+#mixin Mixin
+#{targets: "ic2.core.crop.cropcard.CropRedWheat"}
+zenClass MixinCropRedWheat {
+    #mixin ModifyConstant
+    #{
+    #    method: "getGrowthDuration",
+    #    constant: {intValue: 600}
+    #}
+    function growFaster(value as int) as int {
+        return 200;
+    }
+}
+
+#mixin Mixin
+#{targets: "ic2.core.crop.cropcard.CropVenomilia"}
+zenClass MixinCropVenomilia {
+    #mixin ModifyConstant
+    #{
+    #    method: "getGrowthDuration",
+    #    constant: [{intValue: 600}, {intValue: 400}]
+    #}
+    function growFaster(value as int) as int {
+        return value / 4;
+    }
+}
+
+#mixin Mixin
+#{targets: "ic2.core.item.tfbp.Tfbp$TfbpType"}
+zenClass MixinTfbpType {
+    #mixin Static
+    #mixin ModifyConstant
+    #{
+    #    method: "<clinit>",
+    #    constant: [
+    #      {doubleValue: 2000.0},
+    #      {doubleValue: 4000.0},
+    #      {doubleValue: 2500.0},
+    #      {doubleValue: 3000.0},
+    #      {doubleValue: 8000.0}
+    #    ]
+    #}
+    function decreaseConsumption(value as double) as double {
+        return value / 20.0;
+    }
+}
+
+#mixin Mixin
+#{targets: "ic2.core.item.tool.EntityMiningLaser"}
+zenClass MixinEntityMiningLaser {
+    #mixin ModifyConstant
+    #{
+    #    method: "explode",
+    #    constant: {"floatValue": 5.0}
+    #}
+    function modifyExplodeValue0(value as float) as float {
+        return 3.5f;
+    }
+
+    #mixin ModifyConstant
+    #{
+    #    method: "explode",
+    #    constant: {"floatValue": 0.85}
+    #}
+    function modifyExplodeValue1(value as float) as float {
+        return 1.0f;
+    }
+
+    #mixin ModifyConstant
+    #{
+    #    method: "explode",
+    #    constant: {"floatValue": 0.55}
+    #}
+    function modifyExplodeValue2(value as float) as float {
+        return 0.0f;
+    }
+
+    #mixin Redirect
+    #{
+    #    method: "hitEntity",
+    #    at: {
+    #        value: "FIELD",
+    #        target: "Lic2/core/item/tool/EntityMiningLaser;power:F",
+    #        opcode: 180,
+    #        ordinal: 1
+    #    }
+    #}
+    function modifyExplodeValue2(item as EntityMiningLaser) as float {
+        return 5.0f;
+    }
+}
+
+#mixin Mixin
+#{targets: "ic2.core.item.tool.ItemCropnalyzer"}
+zenClass MixinItemCropnalyzer {
+    #mixin Redirect
+    #{
+    #    method: "onItemUseFirst",
+    #    at: {
+    #       value: "INVOKE",
+    #       target: "Lic2/api/crops/CropCard;getGrowthDuration(Lic2/api/crops/ICropTile;)I"
+    #    }
+    #}
+    function showGrowFasterInfo(cropCard as CropCard, tile as ICropTile) as int {
+        return cropCard.getGrowthDuration(tile) / 10;
+    }
+}
+
+#mixin Mixin
+#{targets: "ic2.core.item.tool.ItemToolMiningLaser"}
```

Comment:
Power storage change is applied fine, but power usage change not working. Mining Laser still consume only default, low amount of power, making laser usage OP: 

https://github.com/user-attachments/assets/12ba5c37-9e74-47cc-8b41-5907f48c385d